netmaster/mastercfg/policyState.go

```diff
@@ -293,6 +293,13 @@ func (gp *EpgPolicy) createOfnetRule(rule *contivModel.Rule, dir string) (*ofnet
 		return nil, err
 	}
 
+	// Send AddRule to netplugin agents
+	err = addPolicyRuleState(ofnetRule)
```

Comment:
Up to you whether to follow it or not, but inline error/ok checks are convention in Go code (and in the Go compiler + stdlib's code).  Our projects seem to follow it about half of the time 😄